controller/engine_image_controller.go

```diff
@@ -797,3 +797,36 @@ func (ic *EngineImageController) createEngineImageDaemonSetSpec(ei *longhorn.Eng
 
 	return d, nil
 }
+
+func (ic *EngineImageController) isResponsibleFor(ei *longhorn.EngineImage) bool {
```

Comment:
see comment about other isResponsible function and see if the same issues apply here.

Reply:
Thank you. Fixed